frontend/src/components/Overview.tsx

```diff
@@ -60,8 +60,14 @@ export const OverviewContent: FC<OverviewContentProps> = ({
     )}
     <div className={classes(css.data)}>
       {data.map((column, idx) => (
-        // eslint-disable-next-line react/no-array-index-key
-        <div className={classes(css.column)} key={idx}>
+        <div
+          className={classes(
+            css.column,
+            `${idx + 1 === data.length && css.lastColumn}`,
+          )}
```

Comment:
```suggestion
          className={classes(css.column, {
            [css.borderless]: idx + 1 === data.length || vertical,
          })}
```

This is the better way to add conditional class names. The current one would break as soon as we add a `.false` class (I hope we won't, but still...).

Also, the border seems a bit odd with the vertical layout, so maybe disable it then as well.
Maybe the class could be renamed, as "lastColumn" wouldn't be accurate anymore.